src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Triage and run diagnostics in eight clicks
```

Comment:
that's a bit too marketing-esque & clickbaity I think for docs. but other people might have other thoughts

Reply:
Also note that we say '8 clicks' here, but '10 in nav' and there are 9 steps

Reply:
I kind of agree with zulu, I feel like you can just get rid of the eight clicks part and have a good strong action oriented title with what's left

Reply:
I'm also not quite sure what part of this tutorial is running diagnostics. When I think about that in a tech context, I think about running a script specifically to get some data. An equivalent in APM would be turning on a Thread Profiler. 

However, I think "Triage" is a great word to describe this.

Reply:
this is all very helpful! thanks!